server/utils/utils.ts

```diff
@@ -105,6 +105,16 @@ export const possessive = (string: string): string => {
   return `${string}${string.toLowerCase().endsWith('s') ? '’' : '’s'}`
 }
 
+export const formatStatement = (statement: string): string => {
```

Comment:
I know it was already in the ts code, but this really belongs in the njk somehow.

Reply:
Have moved this to the nunjucks setup file as a filter